src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentPlanCard/PaidPlanCard/PaidPlanCard.jsx

```diff
@@ -1,38 +1,47 @@
-import PropType from 'prop-types'
+import { useParams } from 'react-router-dom'
 
-import { planPropType } from 'services/account'
+import { useAccountDetails, usePlanData } from 'services/account'
 import BenefitList from 'shared/plan/BenefitList'
 import ScheduledPlanDetails from 'shared/plan/ScheduledPlanDetails'
 
 import ActionsBilling from '../shared/ActionsBilling/ActionsBilling'
 import PlanPricing from '../shared/PlanPricing'
 
-function ProPlanCard({ plan, scheduledPhase }) {
-  const seats = plan?.quantity
+function PaidPlanCard() {
+  const { provider, owner } = useParams()
+  const { data: accountDetails } = useAccountDetails({ provider, owner })
+  const { data: planData } = usePlanData({
+    provider,
+    owner,
+  })
+  const scheduledPhase = accountDetails?.scheduleDetail?.scheduledPhase
+  const plan = planData?.plan
+  const marketingName = plan?.marketingName
+  const benefits = plan?.benefits
+  const value = plan?.value
+  const baseUnitPrice = plan?.baseUnitPrice
+  const seats = plan?.planUserCount
```

Comment:
Are you able to provide any default values here if these values are possibly null?

Reply:
So all of these can't be nullable as per our backend/zod schema, so we should be fine here. 

But otherwise, for a scenario that applies, what default value would work here? It almost feels like if we didn't have all of these that we shouldn't show the card, or have designs reflect that state (but to be clear, they're non-nullable here)

<img width="569" alt="Screenshot 2023-11-01 at 9 35 54 AM" src="https://github.com/codecov/gazebo/assets/82913673/2a4109db-c165-4491-8659-dda42bdd2546">